padl/transforms.py

```diff
@@ -93,6 +93,37 @@ def __init__(self, call_info: Optional[inspector.CallInfo] = None,
         self._pd_device = 'cpu'
         self._pd_layers = None
 
+    def _pd_splits(self, input_components=0) -> 'Transform':
```

Comment:
I'm not sure what the function signature should be, but I think something like `Tuple[Union[int, List], Tuple[Transform]]`

Reply:
ah right, I think it should be `Tuple[Union[int, List], Tuple[Transform, Transform, Transform]]`